bin/write_viral_gff.py

```diff
@@ -156,11 +168,15 @@ def write_gff(
                 checkv_type = row["provirus"]
                 checkv_quality = row["checkv_quality"]
                 miuvig_quality = row["miuvig_quality"]
+                kmer_freq = row["kmer_freq"]
+                viral_genes = row["viral_genes"]
                 checkv_info = ";".join(
                     [
                         f"checkv_provirus={checkv_type}",
                         f"checkv_quality={checkv_quality}",
                         f"miuvig_quality={miuvig_quality}",
+                        f"kmer_freq={kmer_freq}",
+                        f"viral_genes={viral_genes}",
                     ]
```

Comment:
Maybe add the 'checkv' prefix to `kmer_freq` and `viral_genes` attribute keys to denote where do they come from.

Reply:
easy, yes